dev-vcs/tig/tig-2.2.recipe

```diff
@@ -4,13 +4,13 @@ Tig functions mainly as a git repository browser. It can also help in the \
 staging changes for commit at chunk level and act as a pager for output \
 different git commands."
 HOMEPAGE="http://jonas.nitro.dk/tig/"
```

Comment:
OK, you can keep http://jonas.nitro.dk/tig/ for HOMEPAGE because it is available again ;-)
(The site was not responding when I wrote the message above.)